ape_etherscan/utils.py

```diff
@@ -3,6 +3,8 @@
     "ethereum": "ETHERSCAN_API_KEY",
     "fantom": "FTMSCAN_API_KEY",
     "optimism": "OPTIMISTIC_ETHERSCAN_API_KEY",
+    "polygon-zkevm": "POLYGON_ZKEVM_API_KEY",
```

Comment:
```suggestion
    "polygon-zkevm": "POLYGONSCAN_ZKEVM_API_KEY",
```

Reply:
or something with `scan` in it?

Reply:
will go with POLYGON_ZKEVM_ETHERSCAN_API_KEY